test/end-to-end/pages/screeners/deathcertificateinenglish.js

```diff
@@ -1,17 +1,20 @@
 'use strict';
 
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/screeners/deathcertificateinenglish');
+const deathCertificateEn = require('app/resources/en/translation/screeners/deathcertificateinenglish');
+const deathCertificateCy = require('app/resources/cy/translation/screeners/deathcertificateinenglish');
+const commonContentEn = require('app/resources/en/translation/common');
+const commonContentCy = require('app/resources/cy/translation/common');
 
-module.exports = async function(answer) {
+module.exports = async function(language = 'en', answer) {
     const I = this;
-
+    const deathCertificateContent = language === 'en' ? deathCertificateEn : deathCertificateCy;
+    const commonContent = language === 'en' ? commonContentEn : commonContentCy;
     await I.checkPageUrl('app/steps/ui/screeners/deathcertificateinenglish');
-    await I.waitForText(content.question);
+    if (language === 'en') {
+        await I.waitForText(deathCertificateContent.question);
+    }
```

Comment:
again, not sure why you'd only want to check the text here on English journeys? It will likely help with test stability for both languages